lib/http.js

```diff
@@ -300,43 +291,6 @@ async function _documentLoader(url) {
   return brDocLoader(url);
 }
 
-async function _getExpectedTarget({req}) {
-  const {body: {id: meterId}} = req;
-  const requestUrl = `https://${req.get('host')}${req.url}`;
-  if(meterId !== requestUrl) {
-    throw new BedrockError(
-      'The request URL does not match the meter ID.',
-      'URLMismatchError', {
```

Comment:
Since we're removing this -- the `DataError` above (around line 110: https://github.com/digitalbazaar/bedrock-meter-http/pull/6/files#diff-a51ebd8b5e5766a58abf907d02dfed5c33e3e6dbecc85a41020c0c5b5b97b540R110) should become a `URLMismatchError`. This also matches what we've done with other services.

Reply:
Updated here 39805ba